packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -592,6 +592,10 @@ export class AppStudioPluginImpl {
     return result;
   }
 
+  public async listCollaborator(ctx: PluginContext): Promise<Result<TeamsAppAdmin[], FxError>> {
+    return ok([]);
```

Comment:
will you update this later?

Reply:
I will implement this API later after design doc is ready. We expect to finish the implement in this week.

Reply:
Yes, the detailed implementation will in another pr